doc/modules/clickhouse.md

```diff
@@ -132,6 +132,29 @@ subject_privacy_length = 16;
 
 You can use obfuscated subjects to group messages with a same subject for example.
 
+
+### Extra columns
+
+You can also diversify the received data by adding an extra columns that differ from default.
```

Comment:
.. extract additional data and add it to a set of extra columns. + please specify the minimum versions for this feature: Rspamd 2.4 and Clickhouse 19.3